### PR TITLE
234 improve README and aminer_install to work with all supported OSes.

### LINKED ISSUE #1293: State supported OS
It is not obvious that the AMiner installation script does not support some OS such as Kali Linux. We should clearly state in the README and possibly in the Getting Started Wiki which OS er supported and which are not.

### PR DESCRIPTION
**Pull request by ernstleierzopf**
# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [ ] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [ ] Issues exist for this PR
- [ ] I added related issues using the "Fixes #<issue-id>"-notations
- [ ] This Pull-Requests merges into the "development"-branch

Fixes #1293 

# Submission specific

- [ ] This PR introduces breaking changes
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

# Describe changes:
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This tool parses log data and allows to define analysis pipelines for anomaly de
 
 ## Requirements
 
-In order to install logdata-anomaly-miner a **Linux system** with **python >= 3.6** is required. **Debian-based** distributions are currently recommended. 
+In order to install logdata-anomaly-miner a **Linux system** with **python >= 3.6** is required. **Debian-based** distributions are currently recommended, but also **Fedora** and **RedHat** are supported.
```
Comment (whotwagner) [inline README.md:9]: Please write: "There is only experimental support for fedora"

Comment (whotwagner) [inline README.md:9]: Replace "Debian-based" distributions with all ubuntu and debian **versions** that we have in the tests.

**Review by whotwagner — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ This tool parses log data and allows to define analysis pipelines for anomaly de
 
 ## Requirements
 
-In order to install logdata-anomaly-miner a **Linux system** with **python >= 3.6** is required. **Debian-based** distributions are currently recommended. 
+In order to install logdata-anomaly-miner a **Linux system** with **python >= 3.6** is required. All **Ubuntu** and **Debian** versions that we have in the tests are currently recommended.
```
Comment (whotwagner) [inline README.md:9]: Please state all supported versions explicitly, so that potential users don't have to read the testfiles

**Review by whotwagner — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ernstleierzopf) [reply README.md:9]: done.

**Review by whotwagner — APPROVED** [review]

**Pull request merged**